### ISSUE TITLE
deploy-bdc-aks Notebook cell 17 connect to master throws exception

### ISSUE THREAD

**Issue by pradm** [microsoft/azuredatastudio#6534]
Issue Type: <b>Bug</b>

An unexpected error occurred while connecting. Please file an issue with the title 'Unexpected Error Occurred while Connecting' and include the log file C:\Users\pradm\AppData\Roaming\azuredatastudio\logs\20190730T231710\renderer1.log

The remderer1.log is empty and I notice the following in main.log

[2019-07-30 23:17:10.248] [main] [info] update#setState idle
[2019-07-30 23:17:40.251] [main] [info] update#setState checking for updates
[2019-07-30 23:17:40.532] [main] [info] update#setState idle
[2019-07-31 00:14:46.380] [main] [error] webContents#will-navigate: Prevented webcontent navigation

The output window for Log(Window) shows the exception

[2019-07-31 00:12:32.455] [renderer1] [info] Creating new connection dashboard://providerName:MSSQL|applicationName:azdata|authenticationType:SqlLogin|database:master|server:104.40.72.232,31433|user:sa|group:C777F06B-202E-4480-B475-FA416154D458
[2019-07-31 00:12:32.455] [renderer1] [info] Adding connection dashboard://providerName:MSSQL|applicationName:azdata|authenticationType:SqlLogin|database:master|server:104.40.72.232,31433|user:sa|group:C777F06B-202E-4480-B475-FA416154D458
[2019-07-31 00:12:32.455] [renderer1] [info] Successfully added connection dashboard://providerName:MSSQL|applicationName:azdata|authenticationType:SqlLogin|database:master|server:104.40.72.232,31433|user:sa|group:C777F06B-202E-4480-B475-FA416154D458
[2019-07-31 00:12:32.834] [renderer1] [error] OnConnectionComplete but no connection found 'connection://providerName:MSSQL|applicationName:azdata|authenticationType:SqlLogin|database:|server:104.40.72.232,31433|user:sa|group:undefined' Connections = [connection://providerName:MSSQL|applicationName:azdata|authenticationType:SqlLogin|database:|server:104.40.72.232,31433|user:sa|group:C777F06B-202E-4480-B475-FA416154D458,dashboard://providerName:MSSQL|applicationName:azdata|authenticationType:SqlLogin|database:master|server:104.40.72.232,31433|user:sa|group:C777F06B-202E-4480-B475-FA416154D458]
[2019-07-31 00:12:32.887] [renderer1] [error] Unexpected error occurred while connecting.: Error: Unexpected error occurred while connecting.
    at ConnectionStatusManager.onConnectionComplete (file:///C:/Users/pradm/AppData/Local/Programs/Azure Data Studio - Insiders/resources/app/out/vs/workbench/workbench.main.js:164842:23)
    at ConnectionManagementService.onConnectionComplete (file:///C:/Users/pradm/AppData/Local/Programs/Azure Data Studio - Insiders/resources/app/out/vs/workbench/workbench.main.js:221879:60)
    at MainThreadDataProtocol.$onConnectionComplete (file:///C:/Users/pradm/AppData/Local/Programs/Azure Data Studio - Insiders/resources/app/out/vs/workbench/workbench.main.js:227181:47)
    at RPCProtocol._doInvokeHandler (file:///C:/Users/pradm/AppData/Local/Programs/Azure Data Studio - Insiders/resources/app/out/vs/workbench/workbench.main.js:232825:27)
    at RPCProtocol._invokeHandler (file:///C:/Users/pradm/AppData/Local/Programs/Azure Data Studio - Insiders/resources/app/out/vs/workbench/workbench.main.js:232810:45)
    at RPCProtocol._receiveRequest (file:///C:/Users/pradm/AppData/Local/Programs/Azure Data Studio - Insiders/resources/app/out/vs/workbench/workbench.main.js:232742:32)
    at RPCProtocol._receiveOneMessage (file:///C:/Users/pradm/AppData/Local/Programs/Azure Data Studio - Insiders/resources/app/out/vs/workbench/workbench.main.js:232670:26)
    at RPCProtocol._protocol.onMessage (file:///C:/Users/pradm/AppData/Local/Programs/Azure Data Studio - Insiders/resources/app/out/vs/workbench/workbench.main.js:232560:52)
    at Emitter.fire (file:///C:/Users/pradm/AppData/Local/Programs/Azure Data Studio - Insiders/resources/app/out/vs/workbench/workbench.main.js:10435:38)
    at deliverMessages (file:///C:/Users/pradm/AppData/Local/Programs/Azure Data Studio - Insiders/resources/app/out/vs/workbench/workbench.main.js:32249:25)
    at source (file:///C:/Users/pradm/AppData/Local/Programs/Azure Data Studio - Insiders/resources/app/out/vs/workbench/workbench.main.js:32255:13)
    at Emitter.fire (file:///C:/Users/pradm/AppData/Local/Programs/Azure Data Studio - Insiders/resources/app/out/vs/workbench/workbench.main.js:10435:38)
    at PersistentProtocol._receiveMessage (file:///C:/Users/pradm/AppData/Local/Programs/Azure Data Studio - Insiders/resources/app/out/vs/workbench/workbench.main.js:32480:37)
    at PersistentProtocol._socketDisposables.push._socketReader.onMessage.msg (file:///C:/Users/pradm/AppData/Local/Programs/Azure Data Studio - Insiders/resources/app/out/vs/workbench/workbench.main.js:32358:83)
    at Emitter.fire (file:///C:/Users/pradm/AppData/Local/Programs/Azure Data Studio - Insiders/resources/app/out/vs/workbench/workbench.main.js:10435:38)
    at ProtocolReader.acceptChunk (file:///C:/Users/pradm/AppData/Local/Programs/Azure Data Studio - Insiders/resources/app/out/vs/workbench/workbench.main.js:32099:37)
    at ProtocolReader._register._socket.onData.data (file:///C:/Users/pradm/AppData/Local/Programs/Azure Data Studio - Insiders/resources/app/out/vs/workbench/workbench.main.js:32068:61)
    at Socket.listener (file:///C:/Users/pradm/AppData/Local/Programs/Azure Data Studio - Insiders/resources/app/out/vs/workbench/workbench.main.js:32637:40)
    at ZoneDelegate.invokeTask (C:\Users\pradm\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules\zone.js\dist\zone-node.js:421:31)
    at Zone.runTask (C:\Users\pradm\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules\zone.js\dist\zone-node.js:188:47)
    at ZoneTask.invokeTask (C:\Users\pradm\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules\zone.js\dist\zone-node.js:496:34)
    at Socket.ZoneTask.invoke (C:\Users\pradm\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules\zone.js\dist\zone-node.js:485:48)
    at Socket.emit (events.js:182:13)
    at addChunk (_stream_readable.js:283:12)
    at readableAddChunk (_stream_readable.js:264:11)
    at Socket.Readable.push (_stream_readable.js:219:10)
    at Pipe.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)

C:\Windows\system32>kubectl get pods --all-namespaces
NAMESPACE       NAME                                    READY   STATUS    RESTARTS   AGE
kube-system     coredns-696c56cf6f-j7mpf                1/1     Running   0          53m
kube-system     coredns-696c56cf6f-mw68s                1/1     Running   0          48m
kube-system     coredns-autoscaler-bc55cb685-bgszl      1/1     Running   0          53m
kube-system     heapster-5678f88989-nkrm5               2/2     Running   0          48m
kube-system     kube-proxy-b48f5                        1/1     Running   0          49m
kube-system     kubernetes-dashboard-7b749f655b-82hks   1/1     Running   0          53m
kube-system     metrics-server-5b7d5c6f8d-ft7wj         1/1     Running   0          53m
kube-system     tunnelfront-549c594b8c-vh6r5            1/1     Running   0          53m
mssql-cluster   appproxy-77xxf                          2/2     Running   0          37m
mssql-cluster   compute-0-0                             3/3     Running   0          37m
mssql-cluster   control-z49sm                           3/3     Running   0          45m
mssql-cluster   controldb-0                             2/2     Running   0          45m
mssql-cluster   data-0-0                                3/3     Running   0          36m
mssql-cluster   data-0-1                                3/3     Running   0          36m
mssql-cluster   gateway-0                               2/2     Running   0          30m
mssql-cluster   logsdb-0                                1/1     Running   0          40m
mssql-cluster   logsui-kwndq                            1/1     Running   0          40m
mssql-cluster   master-0                                3/3     Running   0          36m
mssql-cluster   metricsdb-0                             1/1     Running   0          40m
mssql-cluster   metricsdc-2jd8p                         1/1     Running   0          40m
mssql-cluster   metricsui-sbwmc                         1/1     Running   0          40m
mssql-cluster   mgmtproxy-7g4f2                         2/2     Running   0          40m
mssql-cluster   nmnode-0-0                              1/1     Running   1          30m
mssql-cluster   sparkhead-0                             4/4     Running   0          30m
mssql-cluster   storage-0-0                             4/4     Running   0          36m
mssql-cluster   storage-0-1                             4/4     Running   0          36m

Azure Data Studio version: azuredatastudio-insiders 1.10.0-insider (6db84eefa36a9abf3bcfaa767289ec85f80c95ae, 2019-07-30T07:34:33.861Z)
OS version: Windows_NT x64 10.0.17763

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Xeon(R) CPU E5-2620 v3 @ 2.40GHz (12 x 2394)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>oop_rasterization: disabled_off<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_deferred_display_list: disabled_off<br>skia_renderer: disabled_off<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>viz_display_compositor: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|31.92GB (23.10GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|
</details>Extensions: none

**Comment by alanrenmsft** [comment]
followed up offline and the issue has been resolved.

**Comment by pradm** [comment]
Just updating the solution that worked for this scenario; We updated the pandas package from 0.22.0 to 0.25.0 after launching ADS with elevated privileges and re-ran the statement again which worked. The cluster was stable and none of the PODS had any issues. This occurred when we were using deploy-bdc-aks notebook which was published as a part for CTP 3.2 release and in connect to master the hyperlink [Click here to connect to master SQL Server instance] was shown, once we clicked on the link the above error was shown.
